opm/input/eclipse/EclipseState/Grid/EclipseGrid.hpp

```diff
@@ -30,7 +30,7 @@
 #include <optional>
 #include <stdexcept>
 #include <unordered_set>
-#include <vector>
```

Comment:
This line should also be kept I think...

Reply:
Doh.. Thanks!